proto/deckard_service.proto

```diff
@@ -240,15 +271,13 @@ message EditQueueResponse {
 
 // The queue configuration does not change instantly and can take up to 10 minutes to complete update.
 message QueueConfiguration {
-  /*
-    Number of max elements the queue can have.
-
-    To apply a max elements to a queue, set a value greater than 0.
-    To remove the max elements from a queue, set the value to -1.
-    0 will be always ignored and the queue will not be updated.
-
-    All queues are unlimited by default.
-  */
+  // Number of max elements the queue can have.
```

Comment:
Maybe we should document what the replacement policy is in case max_elements is set